overviewwidget.cpp

```diff
@@ -80,6 +80,11 @@ OverviewWidget::OverviewWidget(OverviewModel *model) :
             this, &OverviewWidget::onItemActivated);
 }
 
+OverviewWidget::~OverviewWidget()
+{
+    delete m_indexUpdater;
```

Comment:
I do not like it when `new` and `delete` are separated by inheritance ;) (possible memory leaks etc.) maybe some smart ptr?